TransformatieRDF.md

```diff
@@ -392,16 +392,16 @@ CONSTRUCT {
 }
 WHERE {
   ?relatieklasse a mim:Relatiesoort.
+  ?relatieklasse mim:modelelementidentificatie ?class.
   ?relatieklasse mim:naam ?relatieklassenaam.
   ?relatieklasse mim:bron ?bezitter.
   ?bezitter mim:naam ?bezittersnaam.
   ?bezitter mim:seeAlso ?subject.
   ?relatieklasse mim:doel ?doelklasse.
   ?doelklasse mim:seeAlso ?object.
-  BIND (t:uri(t:CamelCase(?relatieklassenaam)) as ?class)
   BIND (t:nodeshapeuri(?relatieklassenaam) as ?nodeshape)
   BIND (t:propertyshapeuri(?bezittersnaam,?relatieklassenaam) as ?propertyshape)
-  BIND (t:uri(t:camelCase(?relatieklassenaam)) as ?objectproperty)
+  BIND (t:propertyuri(?relatieklassenaam) as ?objectproperty)
```

Comment:
Deze functie is niet gedefinieerd; ook wordt de functie t:statementuri niet meer gebruikt, dus die kan ook weg.